dynatrace-operator-google-marketplace/schema.yaml

```diff
@@ -15,13 +15,13 @@
 x-google-marketplace:
   schemaVersion: v2
   applicationApiVersion: v1beta1
-  publishedVersion: "0.1.0"
+  publishedVersion: "0.2.1"
   publishedVersionMetadata:
     releaseNote: >-
-      v0.1.0
+      v0.2.1
 
-      Features
-      * Initial release of the Dynatrace Operator
+      Bug fix
+      * Fixed a bug where setting resource limits for routing did not work
```

Comment:
This is technically correct for 0.2.1, but if we go for history, this is the first 0.2.0 release as well for GKE Markeplace, so those release notes would also be relevant.

Reply:
Right - added the features there again